### PR TITLE
Resolves the footer

### PR DESCRIPTION
**Pull request by premalrupnur**
<a href="https://gitpod.io/#https://github.com/Xcov19/mycovidconnect/pull/157"></a>

---
This change is [Reviewable](https://reviewable.io/reviews/xcov19/mycovidconnect/157)

### CODE REVIEW

**Comment by premalrupnur** [comment]
@codecakes  resolved the issue plz review

**Comment by premalrupnur** [comment]
@codecakes  may I close this pr???

File: src/constants/index.js [hunk 1]
```diff
@@ -17,7 +17,9 @@ const {
 	REACT_APP_COOKIEBOT_CONSENT_DECLARATION: COOKIEBOT_CONSENT_DECLARATION,
 	REACT_APP_COOKIEBOT_CONSENT_HEAD: COOKIEBOT_CONSENT_HEAD,
 	REACT_APP_COOKIEBOT_CONSENT_HEAD_DATA_CBID: COOKIEBOT_CONSENT_HEAD_DATA_CBID,
-	REACT_APP_TAWKTO: TAWKTO,
+    REACT_APP_TAWKTO: TAWKTO,
```
Comment (codecakes) [inline src/constants/index.js:20]: fix indentations @premalrupnur

File: src/constants/index.js [hunk 2]
```diff
@@ -34,5 +36,7 @@ export {
 	COOKIEBOT_CONSENT_DECLARATION,
 	COOKIEBOT_CONSENT_HEAD,
 	COOKIEBOT_CONSENT_HEAD_DATA_CBID,
-	TAWKTO,
+    TAWKTO,
```
Comment (codecakes) [inline src/constants/index.js:39]: format indentation correctly

File: src/containers/Search.jsx [hunk 3]
```diff
@@ -6,7 +6,9 @@ import Map from '../components/Map';
 import {
 	GOOGLE_MAPS_API_KEY,
 	UBER_CLIENT_ID,
-	alternativeCityNamesLookup,
+    alternativeCityNamesLookup,
```
Comment (codecakes) [inline src/containers/Search.jsx:9]: indentation, same issue here, check every where else in the changed code @premalrupnur

**Review by codecakes — APPROVED** [review]
@premalrupnur pls make indentations/formatting correctly before merging

**Pull request merged**